src/relations.jl

```diff
@@ -1800,6 +1908,117 @@ function saturation_adjustment_given_pθq(
     end
 end
 
+"""
+    saturation_adjustment_given_pθq_anelastic(
+        param_set,
+        p,
+        θ_liq_ice,
+        q_tot,
+        phase_type,
+        maxiter,
+        tol
+    )
+
+Compute the temperature `T` that is consistent with
+
+ - `param_set` an `AbstractParameterSet`, see the [`Thermodynamics`](@ref) for more details
+ - `p` anelastic reference air pressure
+ - `θ_liq_ice` liquid-ice potential temperature
+ - `q_tot` total specific humidity
+ - `phase_type` a thermodynamic state type
+ - `tol` absolute tolerance for saturation adjustment iterations. Can be one of:
+    - `SolutionTolerance()` to stop when `|x_2 - x_1| < tol`
+    - `ResidualTolerance()` to stop when `|f(x)| < tol`
+ - `maxiter` maximum iterations for non-linear equation solve
+
+by finding the root of
+
+`θ_{liq_ice} - liquid_ice_pottemp_sat(param_set, T, ρ, phase_type, q_tot) = 0`
```

Comment:
```suggestion
`θ_{liq_ice} - liquid_ice_pottemp_given_pressure(param_set, T, ρ, phase_type, q_tot) = 0`
```

We should probably add a note that this saturation adjustment assumes anelastic.

Reply:
yes, please!